### PR TITLE
Remove the default max_tokens for /v1/chat/completions

### PR DESCRIPTION
**Pull request by schoi-habana**
removed the original code that limited max_tokens to 100

### CODE REVIEW

**Comment by mandy-li** [comment]
@schoi-habana , the code change removed default value for max_tokens. If it is null in the query, what would be used to decide the max_new_tokens?

**Comment by schoi-habana** [comment]
when --max_tokens is null in the curl request, the output len is determined by max_total_tokens from the server run command

**Comment by mandy-li** [comment]
Please check if there is anywhere else we set default value for max_tokens. If we want to use max_total_tokens from the server as default, then we should use it for all the templates.

**Comment by schoi-habana** [comment]
removed the default max_tokens from the /v1/completions route

**Review by mandy-li — APPROVED** [review]
LGTM

**Comment by srajabos** [comment]
Fails when using LLAMA 3.3 70B model.
ERROR chat_completions:generate:generate_stream: text_generation_router::infer: router/src/infer.rs:148: one of `max_new_tokens` or `truncate` must be set if a fast tokenizer is not in use

File: benchmark/src/main.rs [hunk 1]
```diff
@@ -147,13 +147,14 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
             tracing::info!("Downloading tokenizer");
 
             // Parse Huggingface hub token
-            let auth_token = std::env::var("HUGGING_FACE_HUB_TOKEN").ok();
+            //let auth_token = std::env::var("HUGGING_FACE_HUB_TOKEN").ok();
```
Comment (mandy-li) [inline benchmark/src/main.rs]: why we need this variable name change?

File: router/src/server.rs [hunk 2]
```diff
@@ -608,7 +609,13 @@ async fn completions(
         ..
     } = req;
 
-    let max_new_tokens = max_tokens.or(Some(100));
+    /*let server_max_out_tokens = info.max_total_tokens - info.max_input_length;
```
Comment (mandy-li) [inline router/src/server.rs]: what are those commented-out code for?

**Review by mandy-li — COMMENTED** [review]
need to clean up

**Comment by schoi-habana** [comment]
cleaning up is done. the tokenizer package was bumped up to resolve the Llama3.3 issue

**Comment by srajabos** [comment]
LGTM. Verified for all required configurations.

**Comment by schoi-habana** [comment]
@regisss please review and help us merge the change

**Review by regisss — APPROVED** [review]
LGTM!

**Pull request merged**